src/Dfc.CourseDirectory.Core/DataManagement/ParsedCsvCourseRow.cs

```diff
@@ -106,6 +106,14 @@ public static ParsedCsvCourseRow FromCsvCourseRow(CsvCourseRow row, IEnumerable<
             _ => (CourseStudyMode?)null
         };
 
+        public static string ResolveDeliveryMode(CourseDeliveryMode value) => value switch
```

Comment:
Could use a rebase; I think this method has changed in `main`

Reply:
Yeah those are in the other course download PR that hasn't been merged yet. Hima is currently testing it